mentorship.html

```diff
@@ -6,7 +6,52 @@
 <div class="page-header">
     <h1>Mentorship Programme</h1>
 </div>
-<div class="container">
-    TODO
-</div>
- <br> <br><br> <br><br> <br>
\ No newline at end of file
+<div class="container home">
```

Comment:
remove home class from here (should be reserved for homepage)

Reply:
Removed home class.

Reply:
Just a note, styling .home is removed but now we have lost some formatting on the page (rows and registration classes were applied before).